src/onegov/org/models/ticket.py

```diff
@@ -366,15 +337,25 @@ def resource(self):
         return query.one()
 
     @cached_property
-    def reservations(self):
+    def reservations_query(self):
```

Comment:
It's probably not worth to make the query a `cached_property`. I would change this to `property`